### PR TITLE
mention tutorial code in docs related to react-router integration

### PR DESCRIPTION
**Pull request by marcellosachs**
I had some trouble getting react-router to work, though after @justin808 pointed me to the tutorial code I was able to figure it out. Think it might be useful for others to see a mention of the tutorial code in the react-router docs.

### CODE REVIEW

File: docs/additional_reading/react_router.md [hunk 1]
```diff
@@ -33,3 +35,11 @@ const RouterApp = (props, location) => {
   );
 };
 ```
+
+For a fleshed out integration of react_on_rails with react-router, check out [React Webpack Rails Tutorial Code](https://github.com/shakacode/react-webpack-rails-tutorial), specifically the files:
+
+[routes](https://github.com/shakacode/react-webpack-rails-tutorial/blob/master/client/app/bundles/comments/routes/routes.jsx)
```
Comment (justin808) [inline docs/additional_reading/react_router.md]: let's use the full path `/client/something` for these

and let's use bullets for formatting

**Comment by marcellosachs** [comment]
@justin808 added the full path to the file references and bullet formatting. Let me know if anything else should be adjusted...

**Comment by justin808** [comment]
LGTM! Thanks!

**Comment by marcellosachs** [comment]
no problem! thanks for pointing me to the tutorial!

**Pull request merged**